packages/app-webdir-ui/src/helpers/dataConverter.js

```diff
@@ -239,7 +241,7 @@ export const staffConverter = ({
           id={asuriteEID}
           profileURL={`${profileURLBase}/profile/${asuriteEID}`}
           key={asuriteEID}
-          imgURL={filledDatum.photo_url.raw}
+          imgURL={`${filledDatum.photo_url.raw}${queryConcat}size=medium&break=${dateVal}`}
```

Comment:
Could we avoid formatting the url in the prop? I don't want to sound picky, but I think it would be an improvement in readability if we create a constant with the absolute value of the URL.

Reply:
Okay. I have pushed up a change to account for this.